### ISSUE TITLE
Update peer dependencies

### ISSUE THREAD

**Issue by kallaspriit** [Quramy/ts-graphql-plugin#39]
The library has the following peer dependencies:

```
"peerDependencies": {
  "graphql": "^0.10.2",
  "typescript": "^2.3.0"
},
```

These should be updated to work with TypeScript 3 and latest graphql, something like:
```
"peerDependencies": {
  "graphql": "^0.13.0 | ^14.0.0",
  "typescript": "^2.3.0 | ^3.0.0"
},
```

**Comment by kallaspriit** [comment]
Is this project not maintained any more?

**Comment by shynome** [comment]
I has the same question Is this project not maintained any more?

**Comment by Quramy** [comment]
I've upgraded peer deps versions and published as v1.2.1
